packages/roosterjs-editor-dom/lib/list/VListItem.ts

```diff
@@ -234,8 +242,13 @@ export default class VListItem {
      * Write the change result back into DOM
      * @param listStack current stack of list elements
      * @param originalRoot Original list root element. It will be reused when write back if possible
+     * @param numberDefinition the number definition of BulletListType or NumberingListType saved in the style metadata
      */
-    writeBack(listStack: Node[], originalRoot?: HTMLOListElement | HTMLUListElement) {
+    writeBack(
+        listStack: Node[],
+        originalRoot?: HTMLOListElement | HTMLUListElement,
+        numberDefinition?: NumberDefinition
```

Comment:
same here

Reply:
You just need to declare numberDefinition in this file as global var, and use it directly from each function